core/src/main/java/com/datastax/oss/driver/internal/core/session/throttling/ConcurrencyLimitingRequestThrottler.java

```diff
@@ -145,6 +145,22 @@ public void signalTimeout(@NonNull Throttled request) {
     }
   }
 
+  @Override
+  public void signalCancel(@NonNull Throttled request) {
```

Comment:
The code for signalTimeout/signalCancel is functionally the same except for the log, but it's probably fine this way (complexity of method is low enough that refactoring into a common method is probably not worth it).